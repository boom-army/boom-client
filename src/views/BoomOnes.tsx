import {
  Accordion,
  AccordionDetails,
  AccordionSummary,
  Box,
  Button,
  Chip,
  FormGroup,
  Grid,
  IconButton,
  Link,
  List,
  Popper,
  Stack,
  TextField,
  Typography,
} from "@mui/material";
import CollectionsIcon from "@mui/icons-material/Collections";
import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
import ShareIcon from "@mui/icons-material/Share";
import Face6Icon from "@mui/icons-material/Face6";
import { ThemeContext } from "../contexts/theme";
import { useContext, useState } from "react";
import { AuctionLabel } from "../components/Auctions/AuctionLabel";

export const BoomOnes = () => {
  const { theme } = useContext(ThemeContext);
  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);

  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
    setAnchorEl(anchorEl ? null : event.currentTarget);
  };

  const open = Boolean(anchorEl);
  const id = open ? "simple-popper" : undefined;
  return (
    <Grid
      container
      spacing={1}
      mt={2}
      px={2}
      sx={{ height: "calc(100vh - 100px)", overflowY: "scroll" }}
    >
      <Grid item xs={12}>
        <Box display={"flex"} justifyContent={"space-between"}>
          <Typography variant="h2" sx={{ lineHeight: 1.5 }}>
            Boom Heroes Hero #420
          </Typography>
          <Link>
            <IconButton
              aria-describedby={id}
              type="button"
              onClick={handleClick}
              sx={{
                backgroundColor: theme.accentColor,
                color: theme.primaryColor,
                "&:hover": {
                  backgroundColor: theme.accentColor,
                  opacity: 0.8,
                },
              }}
            >
              <ShareIcon />
            </IconButton>
            <Popper
              id={id}
              open={open}
              anchorEl={anchorEl}
              placement={"left-start"}
            >
              <Box
                sx={{
                  border: `1px solid ${theme.blue.light}`,
                  borderRadius: "8px",
                  p: 1,
                  mr: 1,
                  bgcolor: theme.background2,
                }}
              >
                The content of the Popper.
              </Box>
            </Popper>
          </Link>
        </Box>
        <Box display="flex" justifyContent={"space-between"} mt={2}>
          <Typography mt={2} variant="h4">
            Family: BoomOnes Collection
          </Typography>
          <Stack mt={1} spacing={1} direction="row">
            <Chip
              label="Creator"
              variant="outlined"
              icon={
                <Face6Icon
                  sx={{ "&.MuiChip-icon": { color: theme.accentColor } }}
                />
              }
            />
            <Chip
              label="Series"
              variant="outlined"
              icon={
                <CollectionsIcon
                  sx={{ "&.MuiChip-icon": { color: theme.accentColor } }}
                />
              }
            />
          </Stack>
        </Box>
      </Grid>
      <Grid item xs={12}>
        <Box sx={{ overflow: "hidden" }}>
          <img src={"/assets/420.png"} width="100%" alt="Boom Heroes #420" />
        </Box>
      </Grid>
      <Grid item xs={12}>
        <Box
          p={2}
          sx={{
            border: `1px solid ${theme.blue.light}`,
            backgroundColor: theme.blue.dark,
          }}
        >
          <Box display={"flex"} justifyContent={"space-between"}>
            <AuctionLabel
              label="Leader"
              content={
                <Chip
                  label="@harkl"
                  variant="outlined"
                  icon={
                    <Face6Icon
                      sx={{ "&.MuiChip-icon": { color: theme.accentColor } }}
                    />
                  }
                />
              }
            />
            <AuctionLabel
              label="Leading bid"
              content={<Typography variant="h3">ðŸ’¥345 ($0.34)</Typography>}
            />
            <Box>
              <AuctionLabel
                label="Time left"
                content={
                  <Typography variant="h3" display={"inline"} ml={0.5}>
                    00:46
                  </Typography>
                }
              />
            </Box>
          </Box>
          <Box pt={4} display="flex" justifyContent={"center"}>
            <FormGroup>
              <TextField
                autoFocus
                id="bid"
                inputProps={{ inputMode: "numeric", pattern: "[0-9]*" }}
                label="Bid"
                name="bid"
                required
                type="number"
                InputLabelProps={{
                  shrink: true,
                  style: { color: theme.secondaryColor },
                }}
                InputProps={{
                  style: { color: theme.secondaryColor },
                }}
                value={345}
                onChange={() => {}}
              />
              <Button
                variant="contained"
                size="large"
                sx={{ minWidth: "50%", marginTop: "1em" }}
              >
                <Typography display={"inline"}>Place </Typography>
                <Typography
                  display={"inline"}
                  sx={{ fontWeight: 600 }}
                  ml={0.5}
                >
                  ðŸ’¥346 ($32.89)
                </Typography>
                <Typography display={"inline"} ml={0.5}>
                  Bid
                </Typography>
              </Button>
            </FormGroup>
          </Box>
          <Box mt={2} display="flex" justifyContent={"center"}>
            <Link
              href="https://dex.aldrin.com/swap?base=USDC&quote=BMA"
              target={"_blank"}
            >
              <Typography variant={"body2"}>
                Don't have any ðŸ’¥BMA? Get some here.
              </Typography>
            </Link>
          </Box>
        </Box>
      </Grid>
      <Grid item xs={12}>
        <Box mt={2}>
          <Accordion expanded={true}>
            <AccordionSummary
              expandIcon={<ExpandMoreIcon />}
              aria-controls="description-content"
              id="description-header"
            >
              <Typography>Description</Typography>
            </AccordionSummary>
            <AccordionDetails>
              <Typography>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                Suspendisse malesuada lacus ex, sit amet blandit leo lobortis
                eget.
              </Typography>
            </AccordionDetails>
          </Accordion>
        </Box>
        <Box>
          <Typography variant="h6">Bid History</Typography>
          <List></List>
        </Box>
        <Box>Metadata</Box>
        <Box>Price</Box>
      </Grid>
      <Grid item xs={12}>
        Promo box
      </Grid>
    </Grid>
  );
};
