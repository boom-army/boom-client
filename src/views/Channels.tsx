import { Box } from "@mui/material";
import React from "react";

export const ChannelView: React.FC = () => {
  return (
    <>
      <Box>Boom</Box>
    </>
  );
};
