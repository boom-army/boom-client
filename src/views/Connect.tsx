import React from "react";
import { Connect } from "../components/Connect";

export const ConnectView = () => {
  return <Connect />;
};
