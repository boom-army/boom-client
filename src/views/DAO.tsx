import React from "react";
import { Box, Button, Grid, Typography } from "@mui/material";

export const DAOView = () => {
  const handleSignUp = () => {};

  return (
    <Grid container justifyContent="center">
      <Grid item xs={9}>
        <Box>
          <Typography variant="h2">
            Have we really been doing DAO's on Solana?
          </Typography>
          <img src="promotional-image.jpg" alt="Promotional Image" />
          <Typography variant="body1">
            Boom is about to solve NFT-based DAO solutions. Aren't you sick of
            third-party add-ons and integrations to make your DAO work? Boom is
            building the ULTIMATE solution for turn-key decentralized governance
            and community collaboration. No more hollering at your homies for
            their Telegram deets. No more trying to figure out how to get your
            DAO to work with your NFTs. Boom is here to make it all work
            seamlessly.
          </Typography>
        </Box>
      </Grid>
      <Grid item xs={9}>
        <Button variant="contained" onClick={handleSignUp}>
          Join the Waiting List
        </Button>
      </Grid>
      <Grid item xs={9}>
        <Typography>
          Buy a Boom Hero and be part of the alpha crew with full access and
          participation in the build
        </Typography>
      </Grid>
      <Grid item xs={9}>
        <Typography>Shit post on Meeper while you wait</Typography>
      </Grid>
      <Grid item xs={9}>
        <Typography>Read up on how our dev is progressing</Typography>
      </Grid>
      <Grid item xs={9}>
        <Typography>Get some $BMA and beat the supply-demand imbalance</Typography>
      </Grid>
      <Grid item xs={9}>
        <Typography>
          VCs get in touch to be part of the funding round
        </Typography>
      </Grid>
    </Grid>
  );
};
