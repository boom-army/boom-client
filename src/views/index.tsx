export { BoomOnes } from "./BoomOnes";
export { ChannelFeed } from "./ChannelFeed";
export { Channels } from "./Channels";
export { PeopleView } from "./People";
export { Explore } from "./Explore";
export { Following } from "./Following";
export { Dashboard } from "./Dashboard";
export { News } from "./News";
export { HeroFeed } from "./HeroFeed";
// export { OGMint } from "./OGMint";
