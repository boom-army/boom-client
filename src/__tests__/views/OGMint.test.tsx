import React from "react";
import { OGMint } from "../../views/OGMint";
import { act, fireEvent, render, screen } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

describe("<OGMint/> component :", () => {
  test("display <OGMint/> component ", async () => {
    let rendered = render(<OGMint />, { wrapper: MemoryRouter });
    expect(rendered).toMatchSnapshot();
  });
});

test("Assertion testing of  <OGMint/> component", () => {
  render(<OGMint />, { wrapper: MemoryRouter });
  expect(screen.getByRole("img", { name: "Boom OG NFT Mint" })).toHaveAttribute(
    "src",
    "/assets/boom-og.png"
  );
  expect(screen.getByRole("img", { name: "1303 OG NFT Card" })).toHaveAttribute(
    "src",
    "/assets/inner.png"
  );
  expect(screen.getByRole("img", { name: "Minting Schedule" })).toHaveAttribute(
    "src",
    "/assets/minting.png"
  );
  expect(
    screen.getByRole("button", { name: "Connect Wallet" })
  ).toBeInTheDocument();
});
