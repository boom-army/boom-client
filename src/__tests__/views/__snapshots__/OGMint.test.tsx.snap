// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OGMint/> component : display <OGMint/> component  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg"
        style="margin-top: 20px;"
      >
        <div
          class="MuiBox-root css-7s0ihh"
        >
          <img
            alt="Boom OG NFT Mint"
            src="/assets/boom-og.png"
            width="300em"
          />
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-7 css-5n8ohs-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-h5fkc8"
            >
              <div
                class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
                style="padding: 24px; background-color: rgb(0, 47, 70); border-radius: 6px; border: 3px solid #005078;"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained css-e8az88 MuiButton-containedPrimary"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Connect Wallet
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-5 css-1odrivk-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-130f8nx"
            >
              <div
                class="track-on-window css-vktagw"
              >
                <div
                  class="MuiBox-root css-kb8rte"
                >
                  <img
                    alt="1303 OG NFT Card"
                    class="inner-element"
                    src="/assets/inner.png"
                    width="120"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            To commemorate the official launch of Boom.Army Beta, we're launching 1303 OG NFTs for everyone who is early. There are 1303 NFTs available, and they will receive a 1303 split share of 20% of our Market Place fees in perpetuity. All White List i3o3...3bos tokens mint at 1.303 SOL discount.
          </p>
        </div>
        <div
          class="MuiBox-root css-h5fkc8"
        >
          <img
            alt="Minting Schedule"
            src="/assets/minting.png"
            width="184"
          />
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root css-11xur9t-MuiPaper-root-MuiTableContainer-root"
          >
            <table
              aria-label="simple table"
              class="MuiTable-root css-quj9j6-MuiTable-root"
            >
              <thead
                class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
              >
                <tr
                  class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-a3yr1h-MuiTableCell-root"
                    scope="col"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-xk1c0y-MuiTypography-root"
                    >
                      Serial
                    </p>
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeMedium css-1o746jm-MuiTableCell-root"
                    scope="col"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-15muos2-MuiTypography-root"
                    >
                      Cost (SOL)
                    </p>
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeMedium css-1o746jm-MuiTableCell-root"
                    scope="col"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-15muos2-MuiTypography-root"
                    >
                      Minting (EST)
                    </p>
                  </th>
                </tr>
              </thead>
              <tbody
                class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
              >
                <tr
                  class="MuiTableRow-root css-34nofg-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                    scope="row"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      #1 - 100 | Legendary
                    </p>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      1.303
                    </p>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      Pre-sale: SOLD OUT
                    </p>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-34nofg-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                    scope="row"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      #101 - 250 | Ultra Rare
                    </p>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      1.6
                    </p>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      LOCKED
                    </p>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-34nofg-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                    scope="row"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      #251 - 500 | Rare
                    </p>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      2
                    </p>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      LOCKED
                    </p>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-34nofg-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                    scope="row"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      #501 - 750 | Rare
                    </p>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      2.3
                    </p>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      LOCKED
                    </p>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-34nofg-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                    scope="row"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      #751 - 1000 | Regular
                    </p>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      2.5
                    </p>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      LOCKED
                    </p>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-34nofg-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                    scope="row"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      #1001 - 1303 | Regular
                    </p>
                  </th>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      3
                    </p>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                    >
                      LOCKED
                    </p>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg"
      style="margin-top: 20px;"
    >
      <div
        class="MuiBox-root css-7s0ihh"
      >
        <img
          alt="Boom OG NFT Mint"
          src="/assets/boom-og.png"
          width="300em"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-7 css-5n8ohs-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-h5fkc8"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
              style="padding: 24px; background-color: rgb(0, 47, 70); border-radius: 6px; border: 3px solid #005078;"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained css-e8az88 MuiButton-containedPrimary"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  Connect Wallet
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-5 css-1odrivk-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-130f8nx"
          >
            <div
              class="track-on-window css-vktagw"
            >
              <div
                class="MuiBox-root css-kb8rte"
              >
                <img
                  alt="1303 OG NFT Card"
                  class="inner-element"
                  src="/assets/inner.png"
                  width="120"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          To commemorate the official launch of Boom.Army Beta, we're launching 1303 OG NFTs for everyone who is early. There are 1303 NFTs available, and they will receive a 1303 split share of 20% of our Market Place fees in perpetuity. All White List i3o3...3bos tokens mint at 1.303 SOL discount.
        </p>
      </div>
      <div
        class="MuiBox-root css-h5fkc8"
      >
        <img
          alt="Minting Schedule"
          src="/assets/minting.png"
          width="184"
        />
      </div>
      <div
        class="MuiBox-root css-0"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root css-11xur9t-MuiPaper-root-MuiTableContainer-root"
        >
          <table
            aria-label="simple table"
            class="MuiTable-root css-quj9j6-MuiTable-root"
          >
            <thead
              class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
            >
              <tr
                class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-a3yr1h-MuiTableCell-root"
                  scope="col"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body2 css-xk1c0y-MuiTypography-root"
                  >
                    Serial
                  </p>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeMedium css-1o746jm-MuiTableCell-root"
                  scope="col"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body2 css-15muos2-MuiTypography-root"
                  >
                    Cost (SOL)
                  </p>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeMedium css-1o746jm-MuiTableCell-root"
                  scope="col"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body2 css-15muos2-MuiTypography-root"
                  >
                    Minting (EST)
                  </p>
                </th>
              </tr>
            </thead>
            <tbody
              class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
            >
              <tr
                class="MuiTableRow-root css-34nofg-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                  scope="row"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    #1 - 100 | Legendary
                  </p>
                </th>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    1.303
                  </p>
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    Pre-sale: SOLD OUT
                  </p>
                </td>
              </tr>
              <tr
                class="MuiTableRow-root css-34nofg-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                  scope="row"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    #101 - 250 | Ultra Rare
                  </p>
                </th>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    1.6
                  </p>
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    LOCKED
                  </p>
                </td>
              </tr>
              <tr
                class="MuiTableRow-root css-34nofg-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                  scope="row"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    #251 - 500 | Rare
                  </p>
                </th>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    2
                  </p>
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    LOCKED
                  </p>
                </td>
              </tr>
              <tr
                class="MuiTableRow-root css-34nofg-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                  scope="row"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    #501 - 750 | Rare
                  </p>
                </th>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    2.3
                  </p>
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    LOCKED
                  </p>
                </td>
              </tr>
              <tr
                class="MuiTableRow-root css-34nofg-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                  scope="row"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    #751 - 1000 | Regular
                  </p>
                </th>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    2.5
                  </p>
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    LOCKED
                  </p>
                </td>
              </tr>
              <tr
                class="MuiTableRow-root css-34nofg-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-54wz5t-MuiTableCell-root"
                  scope="row"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    #1001 - 1303 | Regular
                  </p>
                </th>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    3
                  </p>
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-ogwbw4-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1r998hm-MuiTypography-root"
                  >
                    LOCKED
                  </p>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
